GenumKit/Parsers/StoryboardParser.swift

```diff
@@ -66,6 +71,11 @@ public final class StoryboardParser {
           break;
         }
       }
+      
+      private func appendSegue(segue: Segue) {
```

Comment:
What about using a `Set<Segue>` instead of a `[Segue]` here, so we get that for free and won't need the `appendSegue` private method then, letting that work to the `Set`?